frontend/src/components/OneDayCalendar.tsx

```diff
@@ -0,0 +1,29 @@
+import { Calendar, CalendarChangeEvent } from "primereact/calendar";
+
+interface OneDayCalendarProps {
+  isModifiable: boolean;
+  date: Date;
+  setDate: (date: Date) => void;
+}
+
+export default function OneDayCalendar(props: OneDayCalendarProps) {
+  return (
+    <div>
+      <label
+        htmlFor="date"
+        className="p-component text-teal-900 p-text-secondary my-auto pr-2"
+      >
+        Date
+      </label>
+      <Calendar
+        id="date"
+        disabled={!props.isModifiable}
```

Comment:
The not is kind of odd. Here would much rather have a disabled field show through with that being optional